var/spack/repos/builtin/packages/texinfo/package.py

```diff
@@ -19,7 +19,7 @@ class Texinfo(AutotoolsPackage, GNUMirrorPackage):
     homepage = "https://www.gnu.org/software/texinfo/"
     gnu_mirror_path = "texinfo/texinfo-6.0.tar.gz"
 
-    executables = ["^info$"]
+    executables = ["^info$", "^makeinfo$"]
```

Comment:
You are correct in that this property is used by `spack external find` but for determining which version the external package is so only one is needed.  (See https://spack.readthedocs.io/en/latest/packaging_guide.html#minimal-detection.)

If the goal is to require `makeinfo` be installed, then what you  want is more likely the `sanity check` feature (https://spack.readthedocs.io/en/latest/packaging_guide.html#adding-sanity-checks).  You just want to make sure that all listed files/directories are required of an installation of *all* versions.

Reply:
Thanks, I should have done that.  I also took the occasion to add newer versions of `texinfo`.